modules/accountplan/view/schemeid.php

```diff
@@ -9,14 +9,17 @@
 
 
 $availableSchemeTypes = $schemeControl->listTypes();
-function createSchemeOptions($selected) {
+function createSchemeOptions($selected, $country) {
     global $availableSchemeTypes;
     $schemeTypeOptions = "<option value='0'></option>";
 
     foreach($availableSchemeTypes as $type) {
+        $tmp_str = explode(':', $type['SchemeType']);
+        $scheme_type_country = $tmp_str[0];
         $schemeTypeOptions .= 
-            sprintf("<option value='%d' %s>%s</option>\n",
+            sprintf("<option value='%d' class='%s' %s>%s</option>\n",
                     $type['FakturabankSchemeID'],
+                    ($country == $scheme_type_country || $scheme_type_country == 'FAKTURABANK' ? "active" : "inactive"),
```

Comment:
Check if it is the current selected as well. If I select something and change country later, still display the option I selected.